addons/dragon/functions/fnc_attackProfile_DRAGON.sqf

```diff
@@ -0,0 +1,75 @@
+#include "script_component.hpp"
+/*
+ * Author: Brandon (TCVM) (Code inspired by NouberNou's Dragon Guidance)
+ * Attack profile: Dragon Guidance
+ *
+ * Arguments:
+ * 0: Seeker Target PosASL <ARRAY>
+ * 1: Guidance Arg Array <ARRAY>
+ * 2: Attack Profile State <ARRAY>
+ *
+ * Return Value:
+ * Missile Aim PosASL <ARRAY> - Unused
+ *
+ * Example:
+ * [[1,2,3], [], []] call ace_dragon_fnc_attackProfile_DRAGON;
+ *
+ * Public: No
+ */
+params ["_seekerTargetPos", "_args", "_attackProfileStateParams"];
+_args params ["_firedEH", "", "", "", "_stateParams"];
+_firedEH params ["_shooter","_weapon","","","","","_projectile"];
+_attackProfileStateParams params ["_maxCorrectableDistance", "_wireCut", "_seekerMaxRangeSqr", "_seekerMinRangeSqr", "_wireCutSource", "_lastTime", "_serviceInterval", "_serviceChargeCount", "_serviceChargeAcceleration", "_dragonSpeed"];
+
```

Comment:
probably want to handle null/not alive shooter
`if (!alive _shooter) exitWith { _retPos  };`